### ISSUE TITLE
Spacing

### ISSUE THREAD

**Issue by eaerth** [char101/tabgroupsbtn#3]
Hello again! This isn't necessarily a bug; just a minor hindrance. I was wondering what you had the spacers set at in the beginning and end for the menu. I moved it from the end of the search bar to the beginning of my tab toolbar and I love it like that. Though in doing this, it defaults to the very edge of the toolbar which isn't much of a problem if you add spacers but in the recent versions of Firefox they've gotten rid of the spacers so I had to install the Classic Theme Restorer add-on to use them hahaha I've added two of them in the front to make it centered and so I can read the tab group label easier. Other than that though, I guess I'd prefer not to use the theme restorer, just to keep the add-ons to a minimum. Do you plan on adjusting the spacing at all, so that maybe there is a little bit of cushioning before and a bit less at the end? Not sure if it would mess with the layout of its default positioning or not. I really don't understand why they got rid of the spacers or the separators to begin with, I still used those things! haha

Cheers! And thanks a lot for fixing the persistence with the toolbar as well.

**Comment by char101** [comment]
I do not add any spacer to the widget. Also adding a specific spacing is difficult because I do not know where other people will put the widget. It may work in a specific position but look bad in other position.

If you want to customize it without classic theme restorer you can use the stylish addon and write an userchrome stylesheet that adds padding to the widget.

**Comment by eaerth** [comment]
Yeah, that's what I was going to go for next. I'm taking a look at your style sheet now. Cheers!
